vtk/meta.yaml

```diff
@@ -12,12 +12,14 @@ source:
   fn: VTK-5.10.1.win-amd64-py2.7.exe         [win64 and py27]
 
 build:
-  number: 1                                  [linux]
+  number: 2                                  [linux]
 
 requirements:
   build:
     - cmake                                  [linux]
     - python
+    - qt
+    - pyqt
```

Comment:
Yeah, it's a matter of style.  Technically, I'm adding two extensions to the VTK build: One needs Qt (only), and the other needs PyQt (and Qt).  I suppose an analogy would be if I had a Python C-extension module that used the Python HDF5 API **and** also used h5py in a few places.  Would I list only `h5py` in that case?